example_gh_test.go

```diff
@@ -153,6 +154,43 @@ func ExampleGQLClient_advanced() {
 	fmt.Println(query)
 }
 
+// Add a star to the cli/go-gh repository using the GQL API.
+func ExampleGQLClient_Mutate_simple() {
+	client, err := gh.GQLClient(nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var m struct {
+		AddStar struct {
+			Starrable struct {
+				Repository struct {
+					Stargazers struct {
+						Nodes []struct {
+							Login string
+						}
+					} `graphql:"stargazers(first: $first)"`
+				} `graphql:"... on Repository"`
+				Gist struct {
+					StargazerCount int
+					Url            string
```

Comment:
The `Url` field should be called `URL` according to Go field naming conventions, but I'd say lets omit it unless we are going to consume the value

Reply:
👍